### DISCUSSION TITLE
CNCF Discussion Presentation

### DISCUSSION THREAD

**Discussion by raravena80** [lima-vm/lima#1023] [Ideas]
@AkihiroSuda does anybody from the project would like to present in the [TAG-Runtime meeting](https://bit.ly/cncf-tag-runtime-meeting-notes)?  Thanks!

**Comment by AkihiroSuda** [comment]
Sure!
When shall I make a presentation?

**Comment by raravena80** [comment]
How about September 1st? 

Thx.

**Reply by AkihiroSuda** [reply]
SGTM, thanks

**Reply by AkihiroSuda** [reply]
@raravena80 Is the date confirmed? Could you update the google doc? 🙏

**Reply by AkihiroSuda** [reply]
@raravena80 I saw another agenda was added for Sep 1 in [the document](https://bit.ly/cncf-tag-runtime-meeting-notes), so I assume my session was postponed.
Sep 29 works for me, but Sep 15 is not as I will be attending Open Source Summit EU.

**Reply by raravena80** [reply]
@AkihiroSuda sorry for the late response. Does Oct 6th work for you?  If yes, can you add it to the agenda.  Thanks!

**Reply by AkihiroSuda** [reply]
Thanks, added Lima to Oct 6
